packages/compiler-core/src/utils.ts

```diff
@@ -45,7 +45,7 @@ export function loadDep(name: string) {
   }
 }
 
-export const parseJS: typeof parse = (code: string, options: any) => {
+export const parseJS: typeof parse = (code: string, options: acorn.Options) => {
```

Comment:
```suggestion
export const parseJS: typeof parse = (code, options) => {
```

We can remove explicit types as types can be inferred from `typeof parse`.

Reply:
@znck Totally right! Made the changes.